index.html

```diff
@@ -11037,6 +11037,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				&lt;input id="startTime" type="text" aria-errormessage="msgID" aria-invalid="true" value="11:30 PM" &gt;
 				&lt;span id="msgID" aria-live="assertive"&gt;&lt;span style="visibility:visible"&gt;Invalid time: the time must be between 9:00 AM and 5:00 PM&lt;/span&gt;&lt;/span&gt;
 				</pre>
+				<p class="note">This example uses <code>aria-live="assertive"</code> to indicate that assistive technologies should immediately announce the error message rather than completing other queued announcements before announcing the error message. This increases the likelihood that users are aware of the error message before they move focus out of the input.</p>
```

Comment:
The first sentence contains 3 instances of "the error message" and 3 instances of either "announce", "announcing", or "announcements", all of which which gives it a bit of a "tongue-twistery" feel.

May I suggest removing 1 instance of each, as follows? I think it would still have the same meaning:

```suggestion
				<p class="note">This example uses <code>aria-live="assertive"</code> to indicate that assistive technologies should immediately announce the error message rather than completing other queued announcements first. This increases the likelihood that users are aware of the error message before they move focus out of the input.</p>
```